### PR TITLE
Hide some of the columns in the Mosviz table viewer by default

### PR DESCRIPTION
**Pull request by astrofrog**
### Description

This PR hides the 'Images', '1D Spectra' and '2D Spectra' columns from the table viewer in the Mosviz application. It requires glue-jupyter v0.10 which is in the process of being released, so the CI should be restarted once that release is available (likely in <10 min)

Screenshot from 2021-09-14 20-56-08

cc @ibusko 

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [ ] Is a milestone set? Milestone is only currently required for PRs related to Imviz MVP.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -23,7 +23,7 @@ install_requires =
     bqplot>=0.12.29
     bqplot-image-gl>=1.4.1
     glue-core>=1.2.1
-    glue-jupyter>=0.8.1
+    glue-jupyter>=0.10
```
Comment (pllim) [inline setup.cfg:26]: This pin must have been bumped like 2-3x today!

Comment (astrofrog) [reply setup.cfg:26]: Indeed!

**Review by rosteen — APPROVED** [review]
Simple enough. Thanks!

**Review by ibusko — APPROVED** [review]
Thanks, Tom, this fixes an important outstanding issue in MOSViz.

**Pull request merged**